heka/sandbox/outputs/redshift.lua

```diff
@@ -125,20 +158,85 @@ local function bulk_load()
     end
 end
 
+function esc_str(v)
+    if v == nil then
+        return "NULL"
+    end
+    if string.len(v) > 65000 then
+        v = "TRUNCATED:" .. string.sub(v, 1, 65000)
```

Comment:
For the JSON fields we may just want to return an empty string, truncated JSON isn't too useful